lib/pbench/server/api/resources/datasets_list.py

```diff
@@ -23,6 +25,8 @@ class DatasetsList(ApiBase):
     API class to list datasets based on PostgreSQL metadata
     """
 
+    PAGE_LIMIT = 20
```

Comment:
20 may be handy for debugging, but it's awfully low to go into `main`. I'm tempted to say this should be a config parameter, in `pbench-server.cfg` and/or in my imaginary future server configuration DB/API... but the default should be a good deal higher than 20.